lib/heimdall_tools/help/sarif_mapper.md

```diff
@@ -0,0 +1,5 @@
+  sarif_mapper translates an Sarif results JSON file into HDF format JSON to be viewable in Heimdall
```

Comment:
```suggestion
  sarif_mapper translates a SARIF result JSON file into HDF format JSON to be viewable in Heimdall
```

Reply:
got it. there are a few of this, I will change them to be the same.